src/forms/input/index.tsx

```diff
@@ -55,11 +57,16 @@ export class Input extends React.PureComponent<InputProps> {
             label,
             placeholder,
             size,
-            type
+            type,
+            style
         } = this.props;
         let classNames = ["clr-control-container", className];
+        let styled: any;
         if (disabled)
             classNames.push("clr-form-control-disabled");
+        styled = {
+             ...style
```

Comment:
I guess we can directly use 'style' prop as style = {style}.

Reply:
Yeah I thought of doing the same way, but I kept in this format in case we need to add other options to styling as we have done for label.